gradle/publish-jars.gradle

```diff
@@ -13,7 +13,7 @@ task sourceJar(type: Jar) {
     classifier 'sources'
 }
 
-task testJar(type: Jar) {
+task myTestJar(type: Jar) {
```

Comment:
I'm... not sure about the logic for task naming, but sure

Reply:
so somehow they're conflicting and it's just a massive PITA, so I just called it something different lol not great but 🤷‍♂

Reply:
renamed to `testJarsForPublishing`